new-lamassu-admin/src/pages/CoinATMRadar/CoinATMRadar.js

```diff
@@ -0,0 +1,180 @@
+import { makeStyles } from '@material-ui/core/styles'
+import React, { useState, memo } from 'react'
+import { useQuery, useMutation } from '@apollo/react-hooks'
+import { gql } from 'apollo-boost'
+
+import { BooleanPropertiesTable } from 'src/components/booleanPropertiesTable'
+import { H4, P, Label2 } from 'src/components/typography'
+import { Button } from 'src/components/buttons'
+import Popper from 'src/components/Popper'
+import { Switch } from 'src/components/inputs'
+import { ReactComponent as HelpIcon } from 'src/styling/icons/action/help/zodiac.svg'
+
+import { mainStyles } from './CoinATMRadar.styles'
+
+const initialValues = {
+  active: false,
+  location: null,
+  commissions: false,
+  supportedCryptocurrencies: false,
+  supportedFiat: false,
+  supportedBuySellDirection: false,
+  limitsAndVerification: false,
+  operatorName: null,
+  operatorPhoneNumber: null,
+  operatorEmail: null
+}
+
+const GET_CONFIG = gql`
+  {
+    config
+  }
+`
+
+const SAVE_CONFIG = gql`
+  mutation Save($config: JSONObject) {
+    saveConfig(config: $config)
+  }
+`
+
+const CoinATMRadar = memo(() => {
+  const [helpPopperAnchorEl, setHelpPopperAnchorEl] = useState(null)
+  const [coinAtmRadarConfig, setCoinAtmRadarConfig] = useState(null)
+
+  // TODO: treat errors on useMutation and useQuery
+  const [saveConfig] = useMutation(SAVE_CONFIG, {
+    onCompleted: configResponse =>
+      setCoinAtmRadarConfig(configResponse.saveConfig.coinAtmRadar)
+  })
+  useQuery(GET_CONFIG, {
+    onCompleted: configResponse => {
+      setCoinAtmRadarConfig(
+        configResponse?.config?.coinAtmRadar ?? initialValues
+      )
+    }
+  })
+
+  const save = it => saveConfig({ variables: { config: { coinAtmRadar: it } } })
+
+  const handleOpenHelpPopper = event => {
+    setHelpPopperAnchorEl(helpPopperAnchorEl ? null : event.currentTarget)
+  }
+
+  const handleCloseHelpPopper = () => {
+    setHelpPopperAnchorEl(null)
+  }
+
+  const mapPropertiesToTableElements = properties =>
+    Object.entries(properties)
+      .map(it => ({
+        name: it['0'],
+        value: it['1']
+      }))
+      .filter(it => it.value != null)
+
+  const useStyles = makeStyles(mainStyles)
+
+  const classes = useStyles()
+  const helpPopperOpen = Boolean(helpPopperAnchorEl)
+
+  if (!coinAtmRadarConfig) return null
```

Comment:
I don't get this one, are we not showing the screen if the config is null? What does it mean to have this configuration null?

Reply:
The config will be null only until useQuery is completed. After that, the page is rendered. As the code is now removing this would cause some errors. Any suggestions to improve this flow?

Reply:
Ohh, got it. It can stay as is.